tsconfig.json

```diff
@@ -18,7 +18,6 @@
     "downlevelIteration": true,
     "experimentalDecorators": true,
     "jsx": "react",
-    "typeRoots": ["node_modules/@types"],
```

Comment:
breaking change from https://devblogs.microsoft.com/typescript/announcing-typescript-5-1-beta/#explicit-typeroots-disables-upward-walks-for-node_modules-types